http/http_connection.hpp

```diff
@@ -442,6 +445,35 @@ class Connection :
         {
             return;
         }
+#ifdef BMCWEB_ENABLE_LINUX_AUDIT_EVENTS
+        if ((req->method() == boost::beast::http::verb::post) ||
+            (req->method() == boost::beast::http::verb::patch) ||
+            (req->method() == boost::beast::http::verb::delete_))
+        {
+            if (!crow::authorization::isOnWhitelist(req->url, req->method()))
```

Comment:
If we need this can starting on line 362 be
```

        if (!crow::authorization::isOnWhitelist(req->url, req->method()) &&
            thisReq.session == nullptr)
        {
            BMCWEB_LOG_WARNING << "[AuthMiddleware] authorization failed";
            forward_unauthorized::sendUnauthorized(
                req->url, req->getHeaderValue("User-Agent"),
                req->getHeaderValue("Accept"), res);
                
            #ifdef BMCWEB_ENABLE_LINUX_AUDIT_EVENTS
                    if ((req->method() == boost::beast::http::verb::post) ||
            (req->method() == boost::beast::http::verb::patch) ||
            (req->method() == boost::beast::http::verb::delete_))
        {
                     if (res.result() != boost::beast::http::status::ok)
                {
                    audit::auditEvent(
                        *req,
                        ("op=" + std::string(req->methodString()) + ":" +
                         std::string(req->target()) + " ")
                            .c_str(),
                        false);
                }       
            }
            #endif
            completeRequest();
            return;
        }
 ```

Reply:
Or what if for now, we just showed completed audit traces? so just remove this 
            if (!crow::authorization::isOnWhitelist(req->url, req->method())) and this                 if (res.result() != boost::beast::http::status::ok)